source/features/command-palette-navigation-shortcuts.tsx

```diff
@@ -20,11 +20,15 @@ function init(): void {
 	delegate(document, 'command-palette', 'keydown', commandPaletteKeydown);
 }
 
-void features.add(import.meta.url, {
-	shortcuts: {
-		'ctrl n': 'Select next item in command palette',
-		'ctrl p': 'Select previous item in command palette',
-	},
-	awaitDomReady: false,
-	init: onetime(init),
-});
+const onMacOs = !!navigator.userAgent.match(/Macintosh/);
```

Comment:
You can use this

https://github.com/refined-github/refined-github/blob/c9d133547f45321c28fb7ddb35ffc08189c9082e/source/github-helpers/index.ts#L73